assets/style.scss.liquid

```diff
@@ -4387,6 +4391,14 @@ body:not(.js-drawer-open) .drawer__content {
   .content-wrapper--title {padding-bottom: 2.5em;}
 }
 
+.hr-no-margin {
+  margin-top:0px;
+  margin-bottom:0px;
+}
```

Comment:
can tabs-margin-div be accomplished by something like .tabs-wrapper h4 selector and then setting a margin on that? I think we could get rid of that div then.